docs/cloud-native-security/cspm-benchmark-rules.asciidoc

```diff
@@ -11,21 +16,21 @@ Each benchmark rule checks to see if a specific type of resource is configured a
 * `Ensure IAM policies that allow full "*:*" administrative privileges are not attached`
 * `Ensure the default namespace is not in use`
 
+When benchmark rules are evaluated, the resulting <<findings-page, findings>> data appears on the <<cloud-posture-dashboard, Cloud Security Posture dashboard>>. 
 
-When benchmark rules are evaluated, the resulting <<findings-page, findings>> data appears on the <<cloud-posture-dashboard, Cloud Security Posture dashboard>>.
+NOTE: Benchmark rules are not editable.
 
-To find the Benchmark Integrations page, go to **Rules -> Benchmark rules**. From there, you can view the benchmark rules associated with an existing integration by clicking the integration name.
+[discrete]
+== Review your benchmarks
 
-[role="screenshot"]
-image::images/benchmark-rules.png[Benchmark rules page]
+To see your active benchmarks, go to **Rules -> Benchmarks**. From there, you can click a benchmark's name to view the benchmark rules associated with it. You can click a benchmark rule's name to see details including information about how to remediate it, and related links. 
 
-You can then click on a benchmark rule's name to see details, including information about how to remediate failures and related links.
+Benchmark rules are enabled by default, but you can disable some of them — at the benchmark level — to suit your environment. This means for example that if you have two integrations using the `CIS AWS` benchmark, disabling a rule for that benchmark affects both integrations. To enable or disable a rule, use the **Enabled** toggle on the right of the rules table. 
```

Comment:
These changes would apply to [this section](https://github.com/elastic/security-docs/pull/4936/files#diff-998081b75b58eb83b92bac882954aabd8f0df6654ddb3d9e53881ee479c05a84R28) as well if you accepted them.

Reply:
The "Enabled" instead of "turned on" is reflective of the UI element (the table column with the setting is called "Enabled" and has a toggle for each rule), so I'm reluctant to avoid using it in this case. Other reviewers what do you think?